packages/twenty-front/src/modules/object-metadata/utils/getObjectMetadataAndViewIdFromPath.ts

```diff
@@ -0,0 +1,15 @@
+const EXTRACT_OBJECT_VIEWID_PLURAL_NAME =
+  /\/objects\/([^/?]+)(?:\?view=([^&]+))?/;
+
+export const getObjectAndViewIdFromPath = (path: string) => {
+  const result = String(path).match(EXTRACT_OBJECT_VIEWID_PLURAL_NAME);
+
+  if (!result) {
+    return null;
+  }
+
+  return {
+    componentId: result[1],
```

Comment:
componentId => objectNamePlural ?

Reply:
Won't require anymore, same reason as we changed to have in QueryParamsViewIdEffect.